homeassistant/components/denonavr/services.yaml

```diff
@@ -9,3 +9,12 @@ get_command:
     command:
       description: Endpoint of the command, including associated parameters.
       example: "/goform/formiPhoneAppDirect.xml?RCKSK0410370"
+dynamic_eq_enable:
+  description: "Enable or disable DynamicEQ."
+  fields:
+    entity_id:
+      description: "Name(s) of the denonavr entities."
+      example: "media_player.living_room_receiver"
+    is_dynamic_eq_enabled:
```

Comment:
This is the new state right? It sounds a bit weird that this is passive voice.

How about `set_dynamic_eq` or just  `dynamic_eq`?

Reply:
Yes true. I also had the impression that it sounds weird 😄 
I just took `volume_mute` as an example. I will change it to `dynamic_eq`